content/intro-to-storybook/angular/en/get-started.md

```diff
@@ -14,44 +14,280 @@ Storybook runs alongside your app in development mode. It helps you build UI com
+    },
+    "transform": {
+      "^.+\\.(ts|html)$": "ts-jest",
+      "^.+\\.js$": "babel-jest"
+    },
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json",
+      "node",
+      ".html"
+    ]
+  }
+```
+
+We need to make some changes to some of the files that `@angular/cli` added when the project was instantiated earlier. More specifically `tsconfig.spec.json`,`tsconfig.json` and `tsconfig.app.json`.
+
+Start by updating your `tsconfig.spec.json` to the following:
+
+```json
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./out-tsc/spec",
+    "module": "commonjs",
+    "allowJs": true,
+    "types": ["jest", "jquery", "jsdom", "node"]
+  },
+  "files": ["src/test.ts", "src/polyfills.ts"],
+  "include": ["src/**/*.spec.ts", "src/**/*.d.ts"]
+}
+```
+
+Then change `tsconfig.json` to:
+
+```json
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "downlevelIteration": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "target": "es2015",
+    "typeRoots": ["node_modules/@types"],
+    "lib": ["es2018", "dom"]
+  },
+  "angularCompilerOptions": {
+    "fullTemplateTypeCheck": true,
+    "strictInjectionParameters": true
+  }
+}
+```
+
+And finally `tsconfig.app.json` with the following:
+
+```json
+{
+  "exclude": ["src/test.ts", "src/**/*.spec.ts", "**/*.stories.ts", "src/jest-config"]
+}
+```
+
+We're almost done, only three more changes are required to finish setting up Jest with Angular and Storybook.
+
+Add the following key and value to your `.storybook/tsconfig.json`:
+
+```json
+{
+  ...
+  "compilerOptions": {
+    "skipLibCheck": true,
+    ....
+  },
+  ...
+}
+
+```
+
+We need a script to run our tests, add the following script to your `package.json` to allow `jest` to run:
+
+```json
+{
+  ....
+  "scripts":{
+    ...
+     "jest": "jest --watch"
+  }
+}
+```
+
+And finally update the test file that was created automatically when the project was initialized, more specifically `src/app/app.component.spec.ts` to:
+
+```typescript
+import { render } from '@testing-library/angular';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  it('should render the component', async () => {
+    const { getByText } = await render(AppComponent);
+    expect(getByText('Welcome'));
+  });
+});
+```
+
+Our three frontend app modalities: automated test (Jest), component development (Storybook), and the app itself.
+
+Now we can quickly check that the various environments of our application are working properly:
+
+```bash
+# Run the test runner (Jest) in a terminal (we will add Jest along the way):
+npm run jest
+
+# Start the component explorer on port 6006:
+npm run storybook
```

Comment:
The storybook command doesn't work : 
```
ModuleBuildError: Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Caching was left unconfigured. Babel's plugins, presets, and .babelrc.js files can be configured
for various types of caching, using the first param of their handler functions:
```

Reply:
Exploring your repo, it sounds like @babel/preset-react is required to make the configuration works.

Reply:
@geromegrignon i think that the problem lies elsewhere i'm going to confirm it, but it seems to be related to `api.cache(true);` usage in the `babel.config.js`. If it's confirmed i'll update the code snippet accordingly

Reply:
It's confirmed, It's due to the `api.cache(true)` statement. That will cause storybook to break the build. I've updated the repo. Going to mark this as resolved.